permissioned-token/program/build.sh

```diff
@@ -0,0 +1,6 @@
+cargo build-bpf
+shank idl -o ../sdk/idl
```

Comment:
This step is not needed as `yarn solita` will run `solita` which will find the `.solitarc.js` and download a matching version of the shank-cli and run it to produce the IDL before generating the code from it.
This makes sure that the `shank` version you use in your program is compatible with the shank-cli version used to extract the IDL.

Reply:
oh cool! learned something new thanks :)